mitmproxy/proxy/mode_specs.py

```diff
@@ -209,6 +209,52 @@ def __post_init__(self) -> None:
         _check_empty(self.data)
 
 
+class WireGuardMode(ProxyMode):
+    default_port = 51820
+    transport_protocol: ClassVar[Literal["tcp", "udp"]] = "udp"
+
+    wireguard_conf: str | None = None
+    wireguard_peers: int | None = None
+
+    # noinspection PyDataclass
+    def __post_init__(self) -> None:
+        # settings are empty or a literal "default":
+        if self.data in ["", "default"]:
```

Comment:
Is there a specific reason for adding an explicit `"default"`? I'd just skip that for brevity.

Reply:
Right. For some reason I thought that there might need to be a distinction between the two, but right now, this doesn't make sense. I'll remove the "default" option.